pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseBrokerRequestHandler.java

```diff
@@ -133,7 +133,7 @@ public BaseBrokerRequestHandler(PinotConfiguration config, BrokerRoutingManager
     _queryQuotaManager = queryQuotaManager;
     _tableCache = tableCache;
     _brokerMetrics = brokerMetrics;
-    _disableGroovy = _config.getProperty(CommonConstants.Broker.DISABLE_GROOVY, false);
+    _disableGroovy = _config.getProperty(CommonConstants.Broker.DISABLE_GROOVY, true);
```

Comment:
Suggest adding a constant for `DEFAULT_DISABLE_GROOVY`

Reply:
added